src/lhttpc_sock.erl

```diff
@@ -27,7 +27,7 @@
 
 %%------------------------------------------------------------------------------
 %%% @private
-%%% @author Oscar Hellstr�m <oscar@hellstrom.st>
+%%% @author Oscar Hellström <oscar@hellstrom.st>
```

Comment:
what changed?

Reply:
Encoding. That's strange looking stuff. I reopened in another IDE and fixed this issue. My git client showed this as:
![image](https://user-images.githubusercontent.com/892959/71370811-f6954d00-25a6-11ea-9f6f-5d5aec752e39.png)